### ISSUE TITLE
webimage_extra_packages Not Working Since Yesterday

### ISSUE THREAD

**Issue by croane** [drud/ddev#2778]
**Describe the bug**
All of a sudden the `webimage_extra_packages` config option breaks and doesn't let the ddev local site start. This was working fine before yesterday, but now it is broken on the two ddev projects I tried starting yesterday + today.

It seems to be some kind of key authentication error from `https://dl.yarnpkg.com/debian/pubkey.gpg`? See the solution I came up with below.

**To Reproduce**
Create a new ddev project. Here is a sample config that includes webimage_extra_packages:

```
name: gabby
type: wordpress
docroot: web
php_version: "7.4"
webserver_type: apache-fpm
router_http_port: "80"
router_https_port: "443"
xdebug_enabled: false
additional_hostnames: []
additional_fqdns: []
mariadb_version: "10.2"
mysql_version: ""
provider: default
use_dns_when_possible: true
composer_version: ""
timezone: America/New_York
webimage_extra_packages: [php-memcache, pv, mmv]
disable_settings_management: true
nfs_mount_enabled: true
```

Run `ddev start`. This is the type of error I see:

![image](https://user-images.githubusercontent.com/7648514/106908418-8703f380-66bc-11eb-8d60-df19f68f6746.png)
![image](https://user-images.githubusercontent.com/7648514/106908462-9125f200-66bc-11eb-9940-7901cf290887.png)

I was able to come up with a temporary workaround. I commented out `webimage_extra_packages` and added this post-start hook:

```
hooks:
 post-start:
   # Install additional packages.
   # pv mmv build-essential
   - exec: curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add - && sudo apt-get update && sudo apt-get install -y --no-install-recommends --no-install-suggests --allow-unauthenticated php-memcache pv mmv
```

This was really strange because these projects were starting fine before yesterday. So they must have updated the .gpg key yesterday and prevented the Linux box from being able to download the packages?

**Comment by croane** [comment]
I forgot to mention I am using the alpha version (but we encountered this issue on the stable release to):
ddev version v1.17.0-alpha3

I am on mac Big Sur with the latest updates.

**Comment by rfay** [comment]
The upstream yarn cert expired; they don't maintain it very well. This was fixed yesterday in ddev v1.17.0-alpha4, please upgrade, thanks.

**Comment by croane** [comment]
Ah, you are right. It is fixed in alpha4. I confirmed that fixes this issue. Thanks!

**Comment by rfay** [comment]
Also fixed in v1.16.6 of course. Glad it's working for you.
